fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraVersioningIT.java

```diff
@@ -492,7 +491,6 @@ public void testCreateVersionOfBinaryWithBody() throws Exception {
         }
     }
 
-    @Ignore("Disable until binary version creation from body implemented")
```

Comment:
There are several other instances of `Disable until binary version creation implemented` in this class. Can they also be re-enabled?

Reply:
Hmm, one of them involves reverting, which isn't implemented (`testDatastreamAutoMixinAndRevert`).  The other one (`testVersionHeaders`) involves the NonRdfSourceDescription, and will probably need some rewriting after the PR to version binary descriptions, which I'm still working on.

Reply:
`testVersionHeaders` is now gone and replaced with `testGetLDPNRMementoHeaders` which should just verify the headers returned on a Memento from a LDP-NR resource.